lib/gui/models/supported-formats.js

```diff
@@ -115,7 +115,7 @@ SupportedFormats.service('SupportedFormatsModel', function() {
    * }
    */
   this.isSupportedImage = (imagePath) => {
-    const extension = path.extname(imagePath).slice(1).toLowerCase();
+    const extension = _.join(_.tail(path.extname(imagePath)), '').toLowerCase();
```

Comment:
IMHO this is _much_ more unreadable. What about using `replace('.', '')` instead?

Also, `.${extension}` is used again at the bottom of the function, so I guess you could maybe do:
```
const extname = path.extname(imagePath);
const extension = extname.replace('.', '').toLowerCase();
```
and then replace `.${extension}` with `extname`. Should there also be some check for `extension` being empty?